src/lib/components/TreeRow.vue

```diff
@@ -21,6 +21,11 @@
           </slot>
         </template>
       </template>
+      <slot
+        name="checkbox" 
+        :checked="reactiveNode.checked" 
+        :toggleCheckbox="($event) => { this.toggleCheckbox(reactiveNode, $event)}"
```

Comment:
Buradaki this parametresini kaldırıp composition api yapısına göre yazmamız daha doğru olur diye düşünüyorum. (Aynı işlem 74. satırda da mevcut)